docs: add step to remove CODEOWNERS in the tutorial (#458)
canonical/sphinx-docs-starter-pack: docs/tutorial/set-up.rst

.. _initial-setup:

Copy the starter pack
=====================

If you're starting a new project, clone the `starter pack repository <https://github.com/canonical/sphinx-docs-starter-pack>`_ and begin your project from there.

> If you're adding documentation to an existing software project, download the following files from the repository and copy them into your existing repository:

Reviewer: ```suggestion
If you're adding documentation to an existing software project, download the following from the starter pack repository to your existing repository:
```

Author: 6d844e2

* the entire :file:`docs` directory
* :file:`.readthedocs.yaml` (configuration for the building on Read the Docs)
* :file:`.wokeignore` (configuration for the Woke tool)
* the entire :file:`.github/workflows` directory

.. _remove-unneeded-files:

Remove the unneeded files
=========================

Next, review the files inherited from the starter pack and remove those that could interfere with your project.

> Remove the files that can't be reused:

Author: My 2c:

I prefer not to send the reader elsewhere so early in a tutorial. I think they'll be capable of finding info about CODEOWNERS if they don't know.

If we rewrote the tutorial, I agree that we should reconsider the order and flow of this, and account for what we assume the user to know.

- :file:`.github/CODEOWNERS`
- :file:`.github/workflows/test-starter-pack.yml`

> Review and remove the GitHub workflows in ``.github/workflows/`` that your project might not need:

Reviewer: I think it's worth clarifying here the situations in which you don't need these files and can safely delete them
